Source: Home brewed
...so called MS Extensions do not support "host" (service name) as a wildcard also for "cifs".
Drop-in will fix this issue.

+++ samba-4.9.4/source3/libads/kerberos_keytab.c	2019-01-18 09:21:08.477919092 +0000
@@ -554,11 +554,11 @@
 		}
 	}
 
-#if 0	/* don't create the CIFS/... keytab entries since no one except smbd
+#if 1	/* don't create the CIFS/... keytab entries since no one except smbd
 	   really needs them and we will fall back to verifying against
 	   secrets.tdb */
 
-	ret = ads_keytab_add_entry(ads, "cifs", false));
+	ret = ads_keytab_add_entry(ads, "cifs", false);
 	if (ret != 0 ) {
 		DEBUG(1, (__location__ ": ads_keytab_add_entry failed while "
 			  "adding 'cifs'.\n"));
